packages/employer-api/src/controllers/claim.controller.ts

```diff
@@ -116,6 +116,54 @@ export const createClaim = async (req: any, res: express.Response) => {
     }
 }
 
+export const createLegacyClaim = async (req: any, res: express.Response) => {
+    try {
+        const bceid_guid = req.kauth.grant.access_token.content?.bceid_user_guid
+        if (bceid_guid === undefined) {
+            return res.status(403).send("Not Authorized")
+        }
+        if (!req.body?.guid || req.body.guid !== bceid_guid) {
+            return res.status(403).send("Forbidden")
+        }
+
+        const prefillFields = {
+            isLegacy: true,
+            container: {
+                businessAddress1: req.body.address,
+                employerCity: req.body.city
+            }
+        }
+
+        // Create a new form draft //
+        const createDraftResult = await formService.createLoginProtectedDraft(
+            req.kauth.grant.access_token,
+            process.env.CLAIM_FORM_ID as string,
+            process.env.CLAIM_FORM_VERSION_ID as string,
+            req.body.formKey,
+            prefillFields
+        )
+        if (createDraftResult?.id) {
+            const insertResult = await insertLegacyClaim(
+                req.body.formKey,
+                req.body.guid,
+                createDraftResult.id,
+                req.body.catchment,
+                req.body.storefront
+            )
```

Comment:
Currently the position title and associated application ID do not get set to any value in the claim record. So is it correct that the form currently contains no information specifying the particular legacy application it's associated with? Maybe we could require that the user enter the position title and an application ID number, in addition to their address, before creating the legacy claim? I think there needs to be enough information to identify the associated application.

Reply:
Confirmed with Erin that this is fine.